app/main.py

```diff
@@ -1,10 +1,10 @@
 def format_linter_error(error: dict) -> dict:
     return {
-        it: [
-            v for k, v in error.items()
-            if k in ["code", "line_number", "column_number", "text"]][i]
-        if it != "source" else "flake8"
-        for i, it in enumerate(["name", "line", "column", "message", "source"])
```

Comment:
Couldn't stop thinking I had to use dict comprehension and complicated everything too much, I now realize it)